bpf/process/bpf_generic_kprobe.c

```diff
@@ -80,6 +80,8 @@ generic_kprobe_start_process_filter(void *ctx)
 		return 0;
 	msg->func_id = config->func_id;
 
+	msg->tid = (__u32)get_current_pid_tgid();
```

Comment:
this might go to generic_process_init, it's called by all probe types on init

Reply:
Ok thanks will do